schemas/spdm.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "definitions": {
+        "Responder": {
+            "title": "Spdm tcp responder configuration",
+            "description": "The configuration used to add remote spdm responder to the system",
+            "type": "object",
+            "properties": {
+                "Hostname": {
+                    "description": "Hostname or IP of spdm tcp responder",
+                    "type": "string"
+                },
+                "Port": {
+                    "description": "Network port spdm tcp responder is listening on",
```

Comment:
May be adding the Spec reference here would be better ? 

`Network port SPDM TCP responder is listening on per DSP0287_1.0.0`